bin/oconf

```diff
@@ -1,80 +1,52 @@
 #!/usr/bin/env node
-var oconf = require('../lib/index');
-
-var argp = require('optimist')
-    .usage('Usage: $0 [--lint|--extract-option <option> [--allow-missing-option][--option-as-json]] <file>')
-    .option('help', {
-        alias: 'h',
-        description: 'This help text.',
-        type: 'boolean'
-    })
-    .option('ignore', {
-        alias: 'i',
-        description: 'Absolute path or glob pattern for ignoring included files.',
-        type: 'string'
-    })
-    .option('lint', {
-        alias: 'l',
-        description: 'Lint the input file and report any errors.',
-        type: 'boolean'
-    })
-    .option('relaxed', {
-        alias: 'r',
-        description: 'Use relaxed JSON format.',
-        type: 'boolean'
-    })
-    .option('extract-option', {
-        alias: 'o',
-        description: 'Extract the value of an option.',
-        type: 'string'
-	})
-    .option('option-as-json', {
-        alias: 'j',
-        description: 'The extracted value will be extracted as JSON.',
-        type: 'boolean'
-    })
-    .option('public', {
-        alias: 'p',
-        description: 'Only values in #public objects will be extracted.',
-        type: 'boolean'
-    })
-    .option('json', { type: 'boolean' })
-    .option('allow-missing-option', {
-        alias: 'm',
-        description: 'Do not fail when no such value is present.',
-        type: 'boolean'
-    })
-    .option('allowmissing', { type: 'boolean' });
-
-var argv = argp.argv;
+const oconf = require('../lib/index');
+const { Command } = require('commander');
+const program = new Command();
+
+program
+    .name('oconf')
+    .usage('[--lint|--extractOption <option> [--allowMissingOption][--optionAsJson]] <file>')
```

Comment:
In `commander`, is this equivalent with the old code?

Or should it be like:
`--extract-option`, `--allow-missing-option`  and `--option-as-json` ?